### PR TITLE
#4782 performance: enhance placing order payload by remove unused pro…

### LINKED ISSUE #4782: 413：request entity too large
## Current behavior

When I have about 20 items in my cart, I get an error when I go to the billing department！
Request URL: https://demo.storefrontcloud.io/api/order
Request Method: POST
Status Code: 413 

## Expected behavior

I have about 20 items in my cart,  checkout 

## Version of Vue Storefront

* [ ] Vue Storefront
* [ ] Vue Storefront Next

## Can you handle fixing this bug by yourself?
* [ ] YES
* [ ] NO

## Which [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html) state this refers to? Info for developer. (doesn't apply to Next)
Pick one option.

* [ ] This is a bug report for test version on https://test.storefrontcloud.io - In this case Developer should create branch from `develop` branch and create Pull Request `2. Feature / Improvement` back to `develop`.
* [ ] This is a bug report for current Release Candidate version on https://next.storefrontcloud.io - In this case Developer should create branch from `release` branch and create Pull Request `3. Stabilisation fix` back to `release`.
* [ ] This is a bug report for current Stable version on https://demo.storefrontcloud.io and should be placed in next stable version hotfix - In this case Developer should create branch from `hotfix` or `master` branch and create Pull Request `4. Hotfix` back to `hotfix`.

## Environment details

* Browser: 
* OS: 
* Node: 
* Code Version:

### PR DESCRIPTION
**Pull request by talalus**
Performance: enhance placing order payload by remove unused product attributes from placing order request.
### Related Issues

closes #4782

### Which Environment This Relates To
Check your case. In case of any doubts please read about [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html)

- [ ] Test version (https://test.storefrontcloud.io) - this is a new feature or improvement for Vue Storefront. I've created branch from `develop` branch and want to merge it back to `develop`
- [ ] RC version (https://next.storefrontcloud.io) - this is a stabilisation fix for Release Candidate of Vue Storefront. I've created branch from `release` branch and want to merge it back to `release`
- [ ] Stable version (https://demo.storefrontcloud.io) - this is an important fix for current stable version. I've created branch from `hotfix` or `master` branch and want to merge it back to `hotfix`

### Upgrade Notes and Changelog
- [x] No upgrade steps required (100% backward compatibility and no breaking changes)
- [ ] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/docs/guide/upgrade-notes/README.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing Vue Storefront sites with this new feature

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: core/pages/Checkout.js [hunk 1]
```diff
@@ -258,11 +258,17 @@ export default {
       }
       return paymentMethod
     },
+    prepareOrderProducts () {
+      return this.$store.state.cart.cartItems.map(product => {
```
Comment (Fifciu) [inline core/pages/Checkout.js:262]: What about making it shorter with?:
```js
return this.$store.state.cart.cartItems.map(product => omit(product, config.orders.excludeFields))
```

https://www.geeksforgeeks.org/lodash-_-omit-method/

**Review by Fifciu — CHANGES_REQUESTED** [review]
- Apply `omit` in the new method.
- Update changelog
- Add info about it to the docs and upgrade notes

File: core/pages/Checkout.js (hunk 1 above)
Comment (talalus) [reply core/pages/Checkout.js:262]: new PR is created https://github.com/vuestorefront/vue-storefront/pull/6219

**Comment by talalus** [comment]
New PR is created https://github.com/vuestorefront/vue-storefront/pull/6219

**Pull request closed without merging**